src/views/Profile.vue

```diff
@@ -528,17 +529,16 @@ export default {
   },
 
   mounted() {
-    this.userRef = db.collection('users').doc(auth.currentUser.uid);
-    this.$bind('userDoc', this.userRef).then(() => {
-      if (!this.userDoc.team) {
+    bindUserAndTeamDocs(this, auth.currentUser.uid, 'userDoc', 'teamDoc')
+      .then(result => {
+        this.userRef = result.user;
+        if (result.team) {
+          this.teamRef = result.team;
```

Comment:
i think you can probably just assign this unconditionally, no? if `result.team` doesn't exist, `this.teamRef` should be undefined, which seems reasonable. or maybe `bindUserAndTeamDocs` should return `{ user: ..., team: null }` in the case when the team is unset.

Reply:
Ok, I'll set it unconditionally.